--- ./scripts/Makefile.am.orig	2015-04-09 23:09:22.000000000 +0100
+++ ./scripts/Makefile.am	2015-07-02 02:45:12.844920000 +0100
@@ -21,7 +21,7 @@ bin_SCRIPTS = \
 	dpkg-source \
 	dpkg-vendor
 
-changelogdir = $(pkglibdir)/parsechangelog
+changelogdir = $(pkglibexecdir)/parsechangelog
 changelog_SCRIPTS = \
 	changelog/debian
 
@@ -123,7 +123,7 @@ do_perl_subst = $(AM_V_GEN) \
 		sed -e "s:^\#![[:space:]]*/usr/bin/perl:\#!$(PERL):" \
 		    -e "s:\$$CONFDIR[[:space:]]*=[[:space:]]*['\"][^'\"]*['\"]:\$$CONFDIR='$(pkgconfdir)':" \
 		    -e "s:\$$ADMINDIR[[:space:]]*=[[:space:]]*['\"][^'\"]*['\"]:\$$ADMINDIR='$(admindir)':" \
-		    -e "s:\$$LIBDIR[[:space:]]*=[[:space:]]*['\"][^'\"]*['\"]:\$$LIBDIR='$(pkglibdir)':" \
+		    -e "s:\$$LIBDIR[[:space:]]*=[[:space:]]*['\"][^'\"]*['\"]:\$$LIBDIR='$(pkglibexecdir)':" \
 		    -e "s:\$$DATADIR[[:space:]]*=[[:space:]]*['\"][^'\"]*['\"]:\$$DATADIR='$(pkgdatadir)':" \
 		    -e "s:\$$PROGVERSION[[:space:]]*=[[:space:]]*['\"][^'\"]*[\"']:\$$PROGVERSION='$(PACKAGE_VERSION)':"
 
