--- ./dpkg-deb/build.c.orig	2015-07-10 19:19:17.042932959 +0100
+++ ./dpkg-deb/build.c	2015-07-10 19:22:43.536320418 +0100
@@ -560,7 +560,7 @@ do_build(const char *const *argv)
     if (chdir(dir))
       ohshite(_("failed to chdir to `%.255s'"), dir);
     execlp(TAR, "tar", "-cf", "-", "--format=gnu", "--null", "--no-unquote",
-                       "-T", "-", "--no-recursion", NULL);
+                       "--no-recursion", "-T", "-", NULL);
     ohshite(_("unable to execute %s (%s)"), "tar -cf", TAR);
   }
   close(p1[0]);
